source/features/view-raw-file.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+const createRawUrl = (href: string) => href.replace(/blob\//, 'raw/');
+
+function init() {
+	const fileActionsGroups: HTMLElement[] = select.all(
+		'.file-header .file-actions > div.BtnGroup'
+	);
+
+	for (const group of fileActionsGroups) {
+		const rawUrl = createRawUrl(group.lastElementChild.getAttribute('href'));
+		group.append(
+			<a href={rawUrl} class="btn btn-sm BtnGroup-item" rel="nofollow">
+				View raw
```

Comment:
GitHub just calls this button Raw, not View raw

Reply:
You're correct 👍 
I've called it "View raw" because of the discussion on the issue. Happy to change it though for the sake of consistency.